docs/documentation/getting-started.md

```diff
@@ -335,93 +341,91 @@ You can then delete the first account, add a new account with the right values,
+    - Select a module from the drop-down menu.
+- Click "Next".
+  - If an existing local module was chosen, a list of local file changes that will be made during the negotiation will be shown. These are the differences between the local version of the module and the version held by the host. The shown actions are the actions necessary to align the local module with the host module.
+  - **Warning:** Any local differences will be removed during the project negotiation. These adjustments will only be done if the "Finish" button is selected. If the session negotiation is aborted at this stage, no local files are changed.
 - Click "Finish".
 
+### Changing user colors
 
-### Leaving a Session
+- Open the IntelliJ settings and navigate to `"Editor" > "Color Scheme" > "Saros"`.
+- Select the color scheme to change the colors for.
+- Expand the user whose colors to change.
+  - Currently, Saros internally has the concept of 5 user colors. These will be assigned to the session participants when a session is started. This means that the other user in a two-user-session will not necessarily have the user color 1.
+- Choose the annotation type to change the color for.
+- Adjust the way the annotation is displayed using the options on the right.
+  - These changes will be previewed in the frame on the bottom.
 
-- Choose the Saros window in the bottom bar.
-- Click on "Leave session" (door icon; the right icon).
-- Select "OK".
-  - If you are the host of the session, the client will subsequently be kicked from the session.
+### Following Another Participant
 
+- Click the "Follow" button (![follow icon](images/icons-i/follow.png)).
+- Select the user to follow from the list and click the entry.
 
-### Disconnecting from the XMPP-Server
+- Alternatively, select another participant in the list of session participants in the session view (on the left) and right-click their name.
+- Click the option "Follow participant".
 
-- Choose the Saros window in the bottom bar.
-- Choose the "Connect" icon (connected plug icon; the left icon).
-- Choose "Disconnect server".
+### Leaving the Follow Mode
+- Click the "Follow" button (![follow icon](images/icons-i/follow.png)).
+- Click the option "Leave follow mode".
 
+### Resolving a Desynchronization
+By default, the synchronization button (![synchronization button off](images/icons-i/in_sync.png)) is disabled.
+If Saros detects that the local content has become out of sync with the host (i.e. differs in any way), it will notify the user and enable the synchronization button (![synchronization button on](images/icons-i/out_sync.png)).
+To resolve the desynchronization:
 
-## Restrictions
+- Click the "Synchronization" button (![synchronization button on](images/icons-i/out_sync.png)). This will open the recovery dialog.
+- Click "Yes".
+  - **WARNING** As stated in the dialog, this will replace the content of the affected files with the content of the file on the host's side.
```

Comment:
affected file(s) with the content of the file(s) ...

Reply:
Done.